src/app/settings/vault-timeout-input.component.html

```diff
@@ -41,5 +41,11 @@
         <small>{{ "minutes" | i18n }}</small>
       </div>
     </div>
+
+    <small *ngIf="!getValidMinutesInput()" class="form-text text-danger">
+      <i class="bwi bwi-error" aria-hidden="true"></i> {{ "vaultCustomTimeoutMinimum" | i18n }}
+    </small>
   </div>
+
+  <i class="bwi bwi-blocked" aria-hidden="true"></i>
```

Comment:
What is this icon? It doesn't show up for me.

Reply:
That's an icon I used for testing and forgot to remove